### DISCUSSION TITLE
Is it possible to publish a new release ?

### DISCUSSION THREAD

**Discussion by Allsimon** [TNG/ArchUnit#1062] [General]
36 commits and 4 months since the previous release (1.0.1)

There are a lot of interesting new stuff, can you do a release please ?

**Comment by codecholeric** [comment]
Sure, we can do one soon, I just have one last thing I'm working on that I would like to get in :slightly_smiling_face:

**Comment by tripleo1** [comment]
Might this help?

https://jitpack.io/p/TNG/ArchUnit

**Reply by nbrugger-tgm** [reply]
Yes but it was already released: https://github.com/TNG/ArchUnit/releases/tag/v1.2.0

**Reply by tripleo1** [reply]
FWIW, I tried (my suggestion) and it didn't work. 🤷🏾

**Reply by hankem** [reply]
@tripleo1, which suggestion didn't work?

Since the original post, [ArchUnit 1.1.0](https://github.com/TNG/ArchUnit/releases/tag/v1.1.0) and [ArchUnit 1.2.0](https://github.com/TNG/ArchUnit/releases/tag/v1.2.0) have been released to Maven Central, see
* https://repo1.maven.org/maven2/com/tngtech/archunit/archunit/1.1.0/
* https://repo1.maven.org/maven2/com/tngtech/archunit/archunit/1.2.0/

**Reply by tripleo1** [reply]
I can use ArchUnit fine from Maven Central, just I couldn't figure out jitpack.

Passing comment, I'm done.

**Reply by codecholeric** [reply]
Unfortunately, my experience with Jitpack hasn't been too positive over the years. When I look at the current logs for ArchUnit, it looks like this issue here 🤷 https://github.com/jitpack/jitpack.io/issues/5662 But no comment on that issue.
I just see that it adds `--add-exports` to a java 8 command which then breaks. But I have no time to further look into that 🤷 
But maybe we could ponder about some way to get quicker releases 🤔 Just don't really have time right now. So far the normal release isn't simple enough that I could do it all the time (e.g. I'm hand-picking the release notes, but I kind of also like that step to go over it and write personal thank-you's to all contributors 🤷 )

**Reply by codecholeric** [reply]
I'm wondering if we could just publish every merge to `main` to GitHub packages? 🤔

**Comment by tripleo1** [comment]
An interesting question for somebody that knows better is why not just
publish to github pages?
IOW, the same thing but with less infrastructure?

**Reply by codecholeric** [reply]
Not sure I understand this 🤔 How would that look like? I only used GH Pages to publish HTML content (e.g. the ArchUnit website and user guide) so far. How would you suggest to publish artifacts there, that can then be used by a standard dependency resolution mechanism? 🤔 (e.g. by Maven or Gradle)
